h2o-hadoop/tests/python/pyunit_gbm_on_hive.py

```diff
@@ -17,24 +27,29 @@ def gbm_on_hive():
     select_query = "select * from airlinestest"
     username = "hive"
     password = ""
+
+    # read from S3
     airlines_dataset_original = h2o.import_file(path="https://s3.amazonaws.com/h2o-public-test-data/smalldata/airlines/AirlinesTest.csv.zip")
+    # read from Hive Distributed
     airlines_dataset = h2o.import_sql_select(connection_url, select_query, username, password)
+    airlines_dataset = adapt_airlines(airlines_dataset)
+    # read from Hive Streaming
+    airlines_dataset_streaming = h2o.import_sql_select(connection_url, select_query, username, password, streaming=True)
+    airlines_dataset_streaming = adapt_airlines(airlines_dataset_streaming)
```

Comment:
would find a better place in h2o-py/tests/testdir_apis/H2O_Module/pyunit_h2oimport_sql_select.py

Reply:
This is testing Hive specifically which is only available in the Hadoop smoke tests. The motivation was Hive.

But you have a good point about also testing the API (the R test is also missing). I don't think the test infrastructure could handle test like that right now. I might be able to make a smoke test of R & Python API using Derby/Java DB.